engine/classes/ElggMenuItem.php

```diff
@@ -504,7 +507,6 @@ public function addItemClass($class) {
 		$this->addClass($this->data['itemClass'], $class);
 	}
 
-	// @codingStandardsIgnoreStart
```

Comment:
any idea why this was in place here??

Reply:
no idea, since this is in our control it should comply with our standards